pkg/kv/kvserver/scheduler.go

```diff
@@ -310,20 +324,24 @@ func (s *raftScheduler) Wait(context.Context) {
 	s.done.Wait()
 }
 
-// SetPriorityID configures the single range that the scheduler will prioritize
-// above others. Once set, callers are not permitted to change this value.
-func (s *raftScheduler) SetPriorityID(id roachpb.RangeID) {
-	for _, shard := range s.shards {
-		shard.Lock()
-		shard.queue.SetPriorityID(id)
-		shard.Unlock()
-	}
+// AddPriorityID adds the given range ID to the set of priority ranges.
+func (s *raftScheduler) AddPriorityID(rangeID roachpb.RangeID) {
+	s.priorityIDs.Store(int64(rangeID), priorityIDsValue)
+}
+
+// RemovePriorityID removes the given range ID from the set of priority ranges.
+func (s *raftScheduler) RemovePriorityID(rangeID roachpb.RangeID) {
+	s.priorityIDs.Delete(int64(rangeID))
 }
```

Comment:
Does this set of IDs need to be dynamic? Is there still a static list that could be configured upon the scheduler creation?

Reply:
Yes, it needs to be dynamic -- the meta ranges can split/merge.